Update extract sample and docu (#234)
avaframe/AvaFrame: docs/moduleAna4Prob.rst

ana4Prob: probAna
==========================

> probAna can be used to derive simple probability maps for the point-wise probability that
> an avalanche will exceed a threshold of a result parameter for a given set of
> avalanche simulations. For example, a map of the probability that an avalanche will exceed a
> peak pressure of 1kPa can be created. 

Reviewer: ```suggestion
probAna is used to derive simple probability maps for the point-wise probability of
an avalanche result parameter exceeding a threshold for a given set of
avalanche simulations. For example, a probability map of an avalanche exceeding a
peak pressure of 1kPa can be created. 
```

> In order to generate this map, a set of multiple avalanche simulations is required. This can be generated with com1DFA
> using parameter variation, different release-, entrainment- or resistance scenarios.
> An example is given in runProbAna.py where for the hockey topography, avalanche simulations
> are performed with varying release thickness and then a probability map is generated that
> ranges from 0-1, which corresponds to a probability of 0-100 %.

Reviewer: ```suggestion
A set of multiple avalanche simulations is required to generate these maps. This can be generated with com1DFA
using parameter variation, different release-, entrainment- or resistance scenarios.
See runProbAna.py for an example: avalanche simulations for the hockey topography 
are performed with varying release thickness and a probability map is generated
ranging from 0-1, which corresponds to a probability of 0-100 %.
```
